blog.html

```diff
@@ -0,0 +1,214 @@
+---
+layout: blog_layout
+title: Blog
+---
+
+<!-- Hero -->
+<div class="container content-space-t-3 content-space-t-lg-4 content-space-b-2">
+  <div class="w-md-75 w-lg-50 text-center mx-auto">
+    <h1 class="display-2">Newsroom</h1>
+    <p class="lead">Aktuelle Beiträge und handverlesene Ressourcen</p>
+  </div>
+</div>
+<!-- End Hero -->
+
+<div class="container mb-10">
+  <!-- Card Grid -->
+  <div class="container mb-7">
+    <div class="row justify-content-md-between align-items-md-center">
+      <div class="col-md-5 mb-5 mb-md-0">
+        <!-- Tags -->
+        <div class="d-md-flex align-items-md-center text-center text-md-start">
+          <span class="d-block me-md-3 mb-2 mb-md-1">Tags:</span>
+          <a class="btn btn-soft-primary btn-xs rounded-pill m-1" href="javascript:;">Business</a>
+          <a class="btn btn-soft-primary btn-xs rounded-pill m-1" href="javascript:;">Health</a>
+          <a class="btn btn-soft-primary btn-xs rounded-pill m-1" href="javascript:;">Environment</a>
+          <a class="btn btn-soft-primary btn-xs rounded-pill m-1" href="javascript:;">Adventure</a>
+        </div>

```

Comment:
Bitte die Tags tatsächlich aus den Tags der Inhalte generieren. Dazu gibt's auch eine Section in den Jekyll Docs: https://jekyllrb.com/docs/posts/